landscape.yml

```diff
@@ -1072,18 +1072,18 @@ landscape:
             twitter: https://twitter.com/Ceph
             crunchbase: https://www.crunchbase.com/organization/ceph-foundation
           - item:
-            name: ChubaoFS
+            name: CubeFS
             homepage_url: https://chubao.io/cgi-sys/suspendedpage.cgi
```

Comment:
This probably should be updated before we merge this in as well.

Reply:
> This probably should be updated before we merge this in as well.
> 
> Edit: https://www.chubao.io/ and http://cubefs.io work, but https://cubefs.io fails because of incorrect certs.
> 
> Not sure which it should be updated to.

@jeefy , thanks for reminding，we fix the www.cubefs.io and https://cubefs.io and can be accessed  now.